### PR TITLE
further edits

### PR DESCRIPTION
**Pull request by dmuelle**
#5983

### CODE REVIEW

File: modules/ROOT/pages/instanton-limitations.adoc [hunk 1]
```diff
@@ -52,7 +52,7 @@ public class StartupServlet extends HttpServlet {
 }
 ----
 
-This `Servlet` example uses `loadOnStartup = 1`, which causes the servlet to be initialized before the checkpoint when you are using the  `afterAppStart` option. The runtime detects this conflict and logs the following message:
+This `Servlet` example uses the `loadOnStartup = 1` attribute, which, when you are using the `afterAppStart` option, initializes the servlet before the checkpoint. The runtime detects this conflict and logs the following message:
```
Comment (tjwatson) [inline modules/ROOT/pages/instanton-limitations.adoc:55]: Is it ok to have that many commas?  Perhaps split this up?

> This `Servlet` example uses the `loadOnStartup = 1` attribute. When you are using the `afterAppStart` option the `loadOnStartup = 1` attribute will cause the `Servlet` to initialize before the checkpoint.

File: modules/ROOT/pages/instanton-limitations.adoc [hunk 2]
```diff
@@ -88,7 +88,7 @@ When the InstantOn application container image is run, the environment variable
 
 ----
 
-In this situation, the `beforeAppStart` checkpoint option can be used. Another option is to use a link:https://download.eclipse.org/microprofile/microprofile-config-3.0/microprofile-config-spec-3.0.html#_dynamic_configsource[Dynamic ConfigSource]. The previous example can be modified to use a dynamic ConfigSource by using the `Provider<String>` type for the `exampleConfig` variable:
+In this situation, use the  `beforeAppStart` checkpoint option. Another option is to use a link:https://download.eclipse.org/microprofile/microprofile-config-3.0/microprofile-config-spec-3.0.html#_dynamic_configsource[Dynamic ConfigSource]. The previous example can be modified to use a dynamic ConfigSource by using the `Provider<String>` type for the `exampleConfig` variable:
```
Comment (tjwatson) [inline modules/ROOT/pages/instanton-limitations.adoc:91]: Have an extra space here before `beforeAppStart`.

File: modules/ROOT/pages/instanton-limitations.adoc (hunk 1 above)
Comment (dmuelle) [reply modules/ROOT/pages/instanton-limitations.adoc:55]: fixed in #6680

File: modules/ROOT/pages/instanton-limitations.adoc (hunk 2 above)
Comment (dmuelle) [reply modules/ROOT/pages/instanton-limitations.adoc:91]: fixed in #6680

**Pull request merged**